client/src/components/Header.css

```diff
@@ -0,0 +1,34 @@
+.header { 
+    z-index: 3;
+    height: 70px;
+    width: 100%;
+    margin: 0px auto;
+    position: fixed;
+    background: rgba(0, 0, 0, 0.53);
+    transition: all 0.2s ease-in-out;
+}
+
+#text-header{
+    padding-top: 20px;
+    color:white;
+    font-size: 20px;
+    text-decoration: none;
+}
```

Comment:
Why is this an ID if it's used in 3 places in the Header.js class? I'd use a CLASS instead.